extensions/dirvish-emerge.el

```diff
@@ -147,6 +148,25 @@ The predicate is consumed by `dirvish-emerge-groups'."
     (and (listp mtime)
          (< (float-time (time-subtract (current-time) mtime)) 86400))))
 
+(dirvish-emerge-define-predicate directories
+  "Matches directories."
+  (eq 'dir (car type)))
+
+(dirvish-emerge-define-predicate files
+  "Matches directories."
+  (eq 'file (car type)))
```

Comment:
Please fix this incorrect docstring, thanks.

Reply:
Done now, thanks!